specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2021-11-01-preview/sqlvm.json

```diff
@@ -1697,7 +1697,8 @@
             "Thursday",
             "Friday",
             "Saturday",
-            "Sunday"
+            "Sunday",
+            "Everyday"
```

Comment:
Hi @raych1, This PR is adding "Everyday" enum value to autoPatchingSettings.dayOfWeek. This value was already an accepted value by SQL VM Resource Provider before we released 2021-11-01 version, however, the enum value as missed during the swagger generation process.

How can we avoid this breaking change flag?

Reply:
It has to get breaking change review board approval.